eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/ArrayListIterate.java

```diff
@@ -1658,15 +1656,7 @@ private ArrayListIterate()
      */
     public static <T> ArrayList<T> sortThis(ArrayList<T> list, Comparator<? super T> comparator)
```

Comment:
seems like we do not have replaceAll() on ArrayListIterate. Do we want to add it as a separate commit?

Reply:
What would replaceAll do on ArrayListIterate?  Delegate to ArrayList?

Reply:
Sorry, I thought it was ArrayListAdapter. My bad.